src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -105,9 +122,4 @@ LogInWithShortLivedAuthTokenPage.propTypes = propTypes;
 LogInWithShortLivedAuthTokenPage.defaultProps = defaultProps;
 LogInWithShortLivedAuthTokenPage.displayName = 'LogInWithShortLivedAuthTokenPage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        account: {key: ONYXKEYS.ACCOUNT},
-    }),
-)(LogInWithShortLivedAuthTokenPage);
+export default compose(withLocalize, withOnyx({account: {key: ONYXKEYS.ACCOUNT}}), withOnyx({session: {key: ONYXKEYS.SESSION}}))(LogInWithShortLivedAuthTokenPage);
```

Comment:
NAB, we can keep in new line for good readability

Reply:
These changes were all from running `npm run prettier`